.changeset/calm-workflows-scope.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/server': minor
+---
+
+Added server-derived ownership for dynamic workflow list, get, and upsert routes.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)